### ISSUE TITLE
Module build breaks

### ISSUE THREAD

**Issue by rickmcgeer** [LivelyKernel/lively.next#563]
In the galyleo-dashboard, I'm seeing the following error in response to `npm run build`:
```
Error when using sourcemap for reporting an error: Can't resolve original location of error.
'rollup' is imported by ../../../../tmp/lively.next/lively.freezer/index.js, but could not be resolved – treating it as an external dependency
'@rollup/plugin-json' is imported by ../../../../tmp/lively.next/lively.freezer/index.js, but could not be resolved – treating it as an external dependency
Error when using sourcemap for reporting an error: Can't resolve original location of error.
/private/tmp/lively.next/lively.next-node_modules/rollup/2.70.2/dist/shared/rollup.js:198
        base = Object.assign(new Error(base.message), base);
                             ^

Error: "/private/tmp/lively.next/lively.lang/worker.js" contains dynamic require statements but it is not within the current dynamicRequireRoot "/tmp/lively.next". You should set dynamicRequireRoot to "/private/tmp/lively.next/lively.lang" or one of its parent directories.
    at error (/private/tmp/lively.next/lively.next-node_modules/rollup/2.70.2/dist/shared/rollup.js:198:30)
    at throwPluginError (/private/tmp/lively.next/lively.next-node_modules/rollup/2.70.2/dist/shared/rollup.js:21891:12)
    at Object.error (/private/tmp/lively.next/lively.next-node_modules/rollup/2.70.2/dist/shared/rollup.js:22614:20)
    at Object.error (/private/tmp/lively.next/lively.next-node_modules/rollup/2.70.2/dist/shared/rollup.js:22068:42)
    at checkDynamicRequire (/tmp/lively.next/lively.next-node_modules/@rollup__SLASH__plugin-commonjs/22.0.0/dist/cjs/index.js:2048:14)
    at Object.enter (/tmp/lively.next/lively.next-node_modules/@rollup__SLASH__plugin-commonjs/22.0.0/dist/cjs/index.js:1609:15)
    at SyncWalkerClass.visit (/tmp/lively.next/lively.next-node_modules/estree-walker/2.0.1/dist/estree-walker.umd.js:66:12)
    at SyncWalkerClass.visit (/tmp/lively.next/lively.next-node_modules/estree-walker/2.0.1/dist/estree-walker.umd.js:103:12)
    at SyncWalkerClass.visit (/tmp/lively.next/lively.next-node_modules/estree-walker/2.0.1/dist/estree-walker.umd.js:103:12)
    at SyncWalkerClass.visit (/tmp/lively.next/lively.next-node_modules/estree-walker/2.0.1/dist/estree-walker.umd.js:96:19) {
  code: 'PLUGIN_ERROR',
  id: '/private/tmp/lively.next/lively.lang/worker.js',
  dynamicRequireRoot: '/tmp/lively.next',
  pos: 1217,
  loc: {
    column: 14,
    file: '/private/tmp/lively.next/lively.lang/worker.js',
    line: 23
  },

```

I'm attaching the build.sh and build.mjs scripts

[Archive.zip](https://github.com/LivelyKernel/lively.next/files/10018096/Archive.zip)

**Comment by linusha** [comment]
I cannot reproduce this. 
I had to use node 18 (`nvm use 18`) to get it to work, which is a lively.next thing and has nothing to do with the dashboard, but I think our README is outdated in that regard.

Afterwards, I cloned the dashboard project into a folder inside of the lively.next directory, switched into that directory and on the toplevel of the dashboard ran `npm run build`, which worked. This is all with a fresh clone of lively.next with which I did nothing but executing `install.sh` once, thus on the latest commit.
